src/modules/authentication/api/Personen-kontext-rolle-fields.response.ts

```diff
@@ -0,0 +1,19 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { OrganisationID } from '../../../shared/types/index.js';
+
+export class PersonenkontextRolleFieldsResponse {
```

Comment:
We prefer constructors for responses (see `UserInfoResponse`).
Remove the default values for the properties and add a constructor